core/src/main/scala/org/apache/spark/deploy/worker/Worker.scala

```diff
@@ -70,7 +70,7 @@ private[deploy] class Worker(
   if (conf.get(config.DECOMMISSION_ENABLED)) {
     logInfo("Registering SIGPWR handler to trigger decommissioning.")
     SignalUtils.register("PWR", "Failed to register SIGPWR handler - " +
-      "disabling worker decommission feature.")(decommissionSelf)
+      "disabling worker decommission feature.")(decommissionSelf(triggeredByMaster = false))
```

Comment:
shall we consistently use one name like `triggeredByWorker`?

Reply:
Ok, sounds good.